pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -103,16 +101,19 @@ stdenv.mkDerivation
     libvirt
     libxml2
     openssl
-    qtbase
-    qtx11extras
+    qt6.qtbase
   ];
 
   nativeBuildInputs = [
     cmake
     git
     pkg-config
     slang
-    wrapQtAppsHook
+    qt6.wrapQtAppsHook
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_PREFIX_PATH=${qt6.qtbase};"
   ];
```

Comment:
This is likely redundant.

Reply:
Removed now I'm using `qt6packages.callPackage` :)